perl-test-pod-coverage.yaml

```diff
@@ -0,0 +1,48 @@
+# Generated from https://git.alpinelinux.org/aports/plain/main/perl-test-pod-coverage/APKBUILD
+package:
+  name: perl-test-pod-coverage
+  version: "1.10"
+  epoch: 0
+  description: Perl - Check for pod coverage in your distribution.
+  copyright:
+    - license: Artistic GPL
```

Comment:
`Artistic GPL` doesn't look like a valid license identifier. Wondering if this was meant to be `GPL-1.0-or-later OR Artistic-1.0-Perl`.

Reply:
I re-checked and it seems to be like this https://metacpan.org/pod/Test::Pod::Coverage#COPYRIGHT-&-LICENSE 

>This program is free software; you can redistribute it and/or modify it under the terms of the Artistic License version 2.0.

What should I modify it to then?

Reply:
Maybe it should be just `Artistic-2.0` like in case of https://github.com/wolfi-dev/os/blob/1955bb0dd6b9db1d9ad5c02dd11550dbe480c0d4/perl-devel-stacktrace.yaml#L8 ?

Reply:
Yes, this copyright notice looks more like `Artistic-2.0`.

Reply:
Done, changed, thanks so much 🥳